portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +33,75 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    ArrayList<String> comments = new ArrayList<String>(); 
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    /*
```

Comment:
I'd recommend removing all your commented out code during a final code cleanup pass before you do a PR. Even though it's a small thing, comments tend to get left behind for years, so it really makes a difference over time.